### PR TITLE
EVA-3141 Set up test

### PR DESCRIPTION
**Pull request by nitin-ebi**

### CODE REVIEW

File: src/__tests__/GenotypesTable.test.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+            let study_id = rows[0].cells[cell_id].textContent
+            let json_value = json_variant.alleleFreqs[study_id]
+            expect(rows[row_id].cells[cell_id].textContent).toBe(json_value == undefined ? "" : "" + json_value)
+        }
+    }
+
+})
+
+
+function rgbToHex(rgb) {
```
Comment (sundarvenkata-EBI) [inline src/__tests__/GenotypesTable.test.js]: If this is **_verbatim_** from StackOverflow, suggest making [attributions](https://meta.stackoverflow.com/a/253622) here...

**Review by sundarvenkata-EBI — APPROVED** [review]
Looks good!

File: src/__tests__/GenotypesFilter.test.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    expect(screen.getByRole('checkbox', {name: 'PRJEB24624'})).toBeChecked()
+    expect(screen.getByRole('checkbox', {name: 'PRJEB24630'})).toBeChecked()
+    expect(screen.getByRole('checkbox', {name: 'PRJEB25439'})).toBeChecked()
+    expect(screen.getByRole('checkbox', {name: 'PRJEB26741'})).toBeChecked()
+    expect(screen.getByRole('checkbox', {name: 'PRJEB27771'})).toBeChecked()
+    expect(screen.getByRole('checkbox', {name: 'PRJEB30116'})).toBeChecked()
+    expect(screen.getByRole('checkbox', {name: 'PRJEB32686'})).toBeChecked()
+    expect(screen.getByRole('checkbox', {name: 'PRJEB38010'})).toBeChecked()
+    expect(screen.getByRole('checkbox', {name: 'PRJEB9799'})).toBeChecked()
+})
```
Comment (tcezard) [inline src/__tests__/GenotypesFilter.test.js]: Is it possible to test user interaction here ? 
i.e. what happens when an assembly is changed ?

**Review by tcezard — COMMENTED** [review]
It looks good but I'm not sure the test data needs to be that big.

File: src/__tests__/GenotypesTable.test.js (hunk 1 above)
Comment (nitin-ebi) [reply src/__tests__/GenotypesTable.test.js]: done

File: src/__tests__/GenotypesFilter.test.js (hunk 2 above)
Comment (nitin-ebi) [reply src/__tests__/GenotypesFilter.test.js]: > It looks good but I'm not sure the test data needs to be that big.

had copied the entire response from prod for initial testing. have truncated the test data now

**Review by tcezard — APPROVED** [review]
Looks good, We're not testing user events yet. but we should do that a s a subsequent ticket.

**Comment by nitin-ebi** [comment]
> Looks good, We're not testing user events yet. but we should do that a s a subsequent ticket.

I was working on it, but facing some issues in asserting the elements in child components. Will raise a separate PR for the user events.

**Pull request merged**